charts/ratify/templates/store.yaml

```diff
@@ -20,4 +20,8 @@ spec:
     {{- if .Values.oras.authProviders.awsEcrBasicEnabled }}
     authProvider:
       name: awsEcrBasic
+    {{- end }}
+    {{- if .Values.oras.cache.enabled }}
+    cacheEnabled: true
```

Comment:
TTL-based cache might be inconsistent with the source in a short time. If users wanna enforce strong consistency, they can disable it. On the other hand, it also makes sense to make it enabled by default since the possiblity of inconsitency is very low and can be controlled by the users.